### ISSUE TITLE
Have you ever tried to train with cifa10

### ISSUE THREAD

**Issue by twmht** [amirhfarzaneh/lsoftmax-pytorch#1]
I tried to train with cifar10 and set margin to 2, but got nan.

learning rate is set to 0.1.

Any idea?

**Comment by amirhfarzaneh** [comment]
@twmht Thank you for noting this. No, I have not tested this on CIFAR yet. I might have some free time this weekend to try it out. I will let you know if I had more insights about this issue and will update the repo accordingly

**Comment by twmht** [comment]
Well. the nan problem solved. but the training accuracy is very low.

It seems that it's hard to converge.

by the way, my network architecture is here https://github.com/kuangliu/pytorch-cifar/blob/master/models/shufflenetv2.py

**Comment by SteveTsui** [comment]
> Well. the nan problem solved. but the training accuracy is very low.
> 
> It seems that it's hard to converge.
> 
> by the way, my network architecture is here https://github.com/kuangliu/pytorch-cifar/blob/master/models/shufflenetv2.py

Have got the same problem as urs. Have u solved it yet?

**Comment by EricWWWW** [comment]
I got a similar problem with cifar10 when m=2/3/4.The accuracy rate is only 10% which is extremely low.It seems that only one class has a very high accuracy rate, and the other nine classes are all 0%.

**Comment by shenbc** [comment]
Mine is nan,too

**Comment by cyizhuo** [comment]
I think 'reset_parameters()' can solve nan problem
